src/otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,4 +1,4 @@
 openvino==2023.0
 openvino-model-api==0.1.5
-otx @ git+https://github.com/sovrasov/openvino_training_extensions/@38a9d7b20e3359d6139616c7127eac98d304c9f6#egg=otx
+otx @ git+https://github.com/openvinotoolkit/training_extensions/@e066a04834952257c1c9384a09f472d13b76b264#egg=otx
```

Comment:
Is this commit OK?

Reply:
this is the current develop head. so I think it's okay. is there any concern?

Reply:
Current head is 6f00907ec67b4d601601e71e790109673ca56d82.
And it does not need to be the develop head but the latest commit in this PR if there is some changes in `otx.api` in this PR.